session/session.go

```diff
@@ -1068,6 +1069,19 @@ func CreateSession(store kv.Storage) (Session, error) {
 	return s, nil
 }
 
+func loadSystemTZ(se *session) (string, error) {
+	sql := `select variable_value from mysql.tidb where variable_name = "system_tz"`
+	rss, errLoad := se.Execute(context.Background(), sql)
+	defer rss[0].Close()
```

Comment:
If errLoad is not nill, the rss maybe empty.

Reply:
fixed.